ISPO InnerSource Summit Presentation Slides (#77)
InnerSourceCommons/ispo-working-group: docs/slides/innersource-summit.md

<!---  Slide 2: Opening Statement --->

![bg right](../assets/innersource-commons.svg)

The InnerSource Program Office (ISPO) Working Group develops guidance and products for enterprise InnerSource teams, not individuals or projects.

<!--
1. Define an ISPO
2. ...
-->
--- 
<!--- Slide 3 --->

For those with InnerSource responsibilities in an OSPO, platform engineering, or other centralized group, this working group is for you.

<!-- KEY MESSAGE 3:  -->

--- 
<!--- Slide 4 --->

But what is an ISPO?

"Person(s) working on improving the internal code collaboration experience at full organization scale"

<!-- KEY MESSAGE 4:  

Technically, ISPO is an acronym. InnerSource Programs Office. 
Often however, the people doing ISPO related work in a company aren't in their own team. 
They are part of an OSPO, part of a centralized engineering team, maybe they are single person working under a head engineering. 
It is variable... and for our purpoeses though, it is just someone with InnerSource responsibilities (however you define that)
be it on an OSPO, platform engineering, or any other centralized group.

-->

--- 

<!--- Slide 5 --->

## Lots of overlap

> OSPO <-> ISPO <-> Platform Engineering <-> Internal Engineering <-> Software Engineering Community of Practice

Reviewer: `Developer Experience` also.

-->

--- 
<!--- Slide 8 --->

Q: WHY is the ISPO working group a subset of InnerSource Commons?

A: Our focus on accelerating the teams that run and support InnerSource across an organization rather than guidance for projects or individuals

<!-- KEY MESSAGE 8:  
WHY: The scope of the InnerSource Programs Office (ISPO) working group is more focused on the teams that run and support InnerSource across an organization rather than guidance for projects or individuals
-->

--- 
<!--- Slide 6 --->

Our goal for this talk is to get people to learn more about the ISPO working group

https://github.com/InnerSourceCommons/ispo-working-group/blob/main/CONTRIBUTING.md

<!-- KEY MESSAGE 6:  

https://github.com/InnerSourceCommons/ispo-working-group/blob/main/CONTRIBUTING.md
-->

--- 
<!--- Slide 7 --->

By participating in our working group 
**YOU** can increase the speed and impact of **YOUR** InnerSource Program Office.

<!-- KEY MESSAGE 7:  
By participating in our working group we can increase the speed and impact of your InnerSource Program Office.

NO eat your brocolli speech. Benefits are to the individual.
-->

--- 
<!--- Slide 9 --->

> Examples of products we are working on specifically for teams that drive InnerSource company-wide:

Reviewer: Show [the GitBook](https://innersourcecommons.gitbook.io/managing-innersource-projects/)

Author: Are you thinking of the Managing InnerSource book cover?

<!-- KEY MESSAGE 17: "We want people to take the work in their company backlog and align it to what is being contributed in this working group.
E.g. in peoples' company backlog there are tickets that talk about them working and building this out in the InnerSource Commons." -->

---
<!---  Slide 18: Specific Action --->

Attend an ISPO working group meeting

Get meeting invites by adding an issue: https://github.com/InnerSourceCommons/ispo-working-group/issues/new

<!-- KEY MESSAGE 18: DO THIS ONE THING RIGHT NOW -->

---
<!---  Slide 19: Benefits of Action --->

The working group has the same benefits as open source. 

Build it together ... work together ... reduce duplication.

<!-- "You should know this already - same benefits as open source.
Build it together ... work together ... reduce duplication.
Run our InnerSource programs as a collective open source project." -->

---
<!---  Slide 19: Closing Statement --->

## Collaborate with us!

- GitHub repository https://github.com/InnerSourceCommons/ispo-working-group 
- Kanban board: https://github.com/orgs/InnerSourceCommons/projects/4 
- Join our slack group: https://innersourcecommons.org/community/#ispo 

> - Get meeting invites: https://github.com/InnerSourceCommons/ispo-working-group/issues/new

Reviewer: GitHub must have made some kind of update because now our "join" template doesn't work: https://github.com/InnerSourceCommons/ispo-working-group/blob/main/.github/ISSUE_TEMPLATE/join-wg.yml

Author: Here's the error in the template.

I haven't used this feature, but maybe the field could be a text field asking for name?

<img width="873" alt="image" src="https://github.com/InnerSourceCommons/ispo-working-group/assets/776901/5af58854-ace7-4709-aa9d-88b3603622f5">

# The InnerSource Program Office (ISPO) Working Group
<!---  Slide 2: Opening Statement --->

![bg right](../assets/innersource-commons.svg)

> The InnerSource Program Office (ISPO) Working Group develops guidance and products for enterprise InnerSource teams, not individuals or projects.

Author: 1. Shortened.
2. Dropped "not individuals or projects" since it carries a negative connotation and is less inclusive.

Even someone not part of an enterprise InnerSource team could benefit from following our working group's progress. With that said, our focus is stated explicitly in this sentence.

```suggestion
The InnerSource Program Office (ISPO) Working Group creates resources for enterprise InnerSource teams.
```

<!--
1. Define an ISPO
2. ...
-->
--- 
<!--- Slide 3 --->

> For those with InnerSource responsibilities in an OSPO, platform engineering, or other centralized group, this working group is for you.
> 

Author: 1. Added an example of other centralized groups.
2. Broke out into a list to make it easier for readers to identify themselves in the slide.

Do we want to add a note here to keep people out of these groups engaged? 

Alternatively, this is a quick "check out" cue to save people time.

```suggestion
If you're in any of these groups, we invite you to join us:

* InnerSource responsibilities in an Open Source Program Office (OSPO)
* Platform Engineering Teams
* Other centralized groups such as Enterprice Architecture
```

<!-- KEY MESSAGE 3:  -->

--- 
<!--- Slide 4 --->

> But what is an ISPO?

Author: Expanding the acronym because those extra characters are cheap. :)

```suggestion
But what is an InnerSource Program Office (ISPO)?
```

"Person(s) working on improving the internal code collaboration experience at full organization scale"

<!-- KEY MESSAGE 4:  

Technically, ISPO is an acronym. InnerSource Programs Office. 
Often however, the people doing ISPO related work in a company aren't in their own team. 
They are part of an OSPO, part of a centralized engineering team, maybe they are single person working under a head engineering. 
It is variable... and for our purpoeses though, it is just someone with InnerSource responsibilities (however you define that)
be it on an OSPO, platform engineering, or any other centralized group.

-->

--- 

<!--- Slide 5 --->

## Lots of overlap
OSPO <-> ISPO <-> Platform Engineering <-> Internal Engineering <-> Software Engineering Community of Practice

You don't formally need an ISPO for this content to be useful

<!-- KEY MESSAGE 5:  
Introduce OSPO vs. ISPO. You don't formally need an ISPO for this content to be useful.

-->

--- 
<!--- Slide 8 --->

Q: WHY is the ISPO working group a subset of InnerSource Commons?

A: Our focus on accelerating the teams that run and support InnerSource across an organization rather than guidance for projects or individuals

> 

Author: 1. Shortened
2. We established what we provide in previous slides.

```suggestion
A: We focus on accelerating teams that run and support InnerSource across an organization rather than individuals.
```

<!-- KEY MESSAGE 8:  
WHY: The scope of the InnerSource Programs Office (ISPO) working group is more focused on the teams that run and support InnerSource across an organization rather than guidance for projects or individuals
-->

--- 
<!--- Slide 6 --->

> Our goal for this talk is to get people to learn more about the ISPO working group

Author: 1. Removed for this talk since people know we're in a talk.
2. Made the object about helping the audience rather than about us getting people to learn.

```suggestion
Our objective is to help you discover the benefits of our ISPO working group.
```

---
marp: true
---

# The InnerSource Program Office (ISPO) Working Group
<!---  Slide 2: Opening Statement --->

![bg right 60%](../assets/innersource-commons.svg)

The InnerSource Program Office (ISPO) Working Group creates resources for enterprise InnerSource teams.

<!--
1. Define an ISPO
2. ...
-->
--- 
<!--- Slide 3 --->
![bg right 60%](../assets/ispo-participants.svg)
If you're in any of these groups, we invite you to join us:

* InnerSource responsibilities in an Open Source Program Office (OSPO)
* Platform Engineering Teams

> * Other centralized groups such as Enterprice Architecture

Reviewer: ```suggestion
* Other centralized groups such as Enterprise Architecture
```
